### DISCUSSION TITLE
Interface won't come up in RHEL 8.4 kickstart

### DISCUSSION THREAD

**Discussion by ozgyilmaz** [rhinstaller/anaconda#4223] [Q&A] [answered]
I define a network interface with VLAN tag in %pre section. Include file is created with success:

`network --ip=<ip_address> --hostname=deneme7 --nameserver=<nameserver1>,<nameserver2> --netmask=<netmask> --gateway=<gateway_address> --device=eno6 --vlanid=3700 --bootproto=static`

This configuration works well in RHEL 7.9. After interface configuration is done the interface comes up and OS is installed from remote source with success.

But this configuration doesnt work in RHEL 8.4 (anaconda version 33.16.4.15). Although the ifcfg configuration file created with success, eno6.3700 interface doesnt come up. I use the anaconda shell and execute the below command to bring the interface up:

`# nmcli con up eno6.3700`

So, is this a bug or am i missing something?

Thank you.

**Comment by ozgyilmaz** [comment]
Same problem in RHEL 8.6 which uses anaconda 33.16.6.7

The VLAN interface wont come up automatically.

I drop to shell in anaconda and execute one of the below commands, then the interface comes up:
- systemctl restart NetworkManager
- nmcli con up eno6.3700 (my interface is eno6 and vlanid is 3700)

RHEL 7.9's anaconda doesnt need such a user intervention, it brings the vlan tagged interface automatically.

**Comment by poncovka** [comment, marked as answer]
@rvykydal, could you answer this issue, please?

**Reply by rvykydal** [reply]
@ozgyilmaz Apologies for the delayed reply. It seems like a bug, could you please file a bugzilla ticket for RHEL 8.6 ? Thank you.

**Reply by ozgyilmaz** [reply]
Hello, thank you. A bugzilla form has been filed by RH case contact Renaud Metrich, here:
https://bugzilla.redhat.com/show_bug.cgi?id=2106684

Also a workaround has been published here:
https://access.redhat.com/solutions/6970595

I've also tested the workaround and it works well.

Thanks again.
